locksmith/__tests__/operations/pricingOperations.test.ts

```diff
@@ -0,0 +1,107 @@
+import { describe, it, expect, vi } from 'vitest'
+import * as pricingOperations from '../../src/operations/pricingOperations'
+
+const lockAddress = '0x551c6ecdf819Dc90c5287971072B4651119accD3'
+const network = 5
+const decimals = 18
+const amount = 1
+
+const mockUsdPricing = {
+  price: 6.94,
+  symbol: 'ETH',
+  amount: 2,
+  decimals,
+}
+
+// eslint-disable-next-line
+var mockWeb3Service = {
+  isLockManager: vi.fn(() => Promise.resolve(false)),
+  getLockContract: vi.fn(() => {
+    return {
+      keyPrice: vi.fn(() => 12),
+      tokenAddress: vi.fn(() => '0x'),
+    }
+  }),
+}
+vi.mock('@unlock-protocol/unlock-js', () => ({
+  Web3Service: function Web3Service() {
+    return mockWeb3Service
+  },
+}))
+
+vi.mock('../../src/operations/pricingOperations')
```

Comment:
It does not make sense to mock `pricingOperations` when you are writing tests for `pricingOperations`... please remove.

When testing you should **only** mock _external_ dependencies.